pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1484,4 +1484,8 @@ self: super: {
     dhall = self.dhall_1_30_0;
   };
 
+  # Stack
+  x509-validation = dontCheck super.x509-validation;
+  tls = dontCheck super.tls;
```

Comment:
@dbaynard Thanks for taking the time to figure this out!  I know it can be tough working with nix.

Could you add a more descriptive error message here, so that the next time someone is looking through this they can hopefully figure out why these two things have been marked `dontCheck`?  Is it because the tests for `x509-validation` and `tls` try to access the network?

Once you update this comment I will merge this PR in.

Reply:
I couldn't work out why the tests were failing, so I opened upstream issues.